docs/export-map.rst

```diff
@@ -0,0 +1,163 @@
+export-map
+==========
+
+.. dfhack-tool::
+    :summary: Export fortress map tile data to a JSON file
+    :tags: dev map
+
+WARNING - This command will cause the game to freeze for minutes depending on
+map size and options enabled.
+
+Exports the fortress map tile data to a JSON file. (does not include items,
+characters, buildings, etc.) Depending on options enabled, there will be a
+``KEY`` table in the JSON with relevant [number ID] values that match a number
+to their object type.
+
+Usage
+-----
+
+::
+
+    export-map [include|exclude] [<options>]
+
+Examples
+--------
+
+``export-map``
+    Exports the fortress map to JSON with ALL data included
+
+``export-map include -m -s -v``
+    Exports the fortress map to JSON with only materials, shape, and variant
+    data included
+
+``export-map exclude --variant --hidden --light``
+    Exports the fortress map to JSON with variant, hidden, and light data
+    excluded
+
+Required
+--------
+
+When you are using options, you must include one of these settings.
+
+``include``
+    Include only the data listed from options to the JSON (whitelist)
+
+``exclude``
+    Exclude only the data listed from options to the JSON (blacklist)
+
+Options
+-------
+
+``help``, ``--help``
+    Shows the help menu
+
```

Comment:
we generally don't document "help" options since the canonical way to get help is `help devel/export-map`